packages/lens-react/lib/components/Datatable/hooks/ThemeStatics.ts

```diff
@@ -2,9 +2,9 @@ import { cn } from "@lens/base/lib/utils";
 
 const VANILLA_THEME = {
   no_data: cn("text-gray-600", "dark:text-gray-400"),
-  wrapper: cn("rounded-md", "border-gray-500 ", "dark:border-gray-100"),
+  wrapper: cn("border-gray-500 border-[#eee] ", "dark:border-gray-100"),
```

Comment:
@ShubhamJoshi-1994 Do we have a tailwind colour class for this?

Reply:
fixed @ShreyDhyani